boards/BMS/Src/App/states/App_AllStates.c

```diff
@@ -80,11 +82,17 @@ static void App_AdvertisePackPower(struct Accumulator *accumulator, struct Tract
 
     const float max_cell_temp = App_Accumulator_GetMaxCellTempDegC(accumulator, &segment, &segment);
     const float available_power =
-        min(App_SharedProcessing_LinearDerating(
+        MIN(App_SharedProcessing_LinearDerating(
                 max_cell_temp, MAX_POWER_LIMIT_W, CELL_ROLL_OFF_TEMP_DEGC, CELL_FULLY_DERATED_TEMP),
             MAX_POWER_LIMIT_W);
 }
 
+void App_AllStates_Init()
+{
+    // Reset accumulator settle count
+    acc_meas_settle_count = 0U;
```

Comment:
We should avoid declaring static variables in functions, since there is no way to reset them between test runs.
There were some bugs in the tests that the settle count would count to 3 on the first test, and then stay 3 for the rest.